src/main/java/magicbees/main/utils/compat/bloodmagic/ItemBloodBaseFrame.java

```diff
@@ -44,7 +44,7 @@ public float getLifespanModifier(IBeeGenome genome, IBeeGenome mate, float curre
 
         @Override
         public float getProductionModifier(IBeeGenome genome, float currentModifier) {
-            return 1f;
+            return 3f; // buffed post additive production change
```

Comment:
Nope, this is meant to stay 1, magic frame's production is 1. Keep them consistent.

If you're changing magic to 2, this should be 2.